### ISSUE TITLE
I can't seem to pass the tests after install

### ISSUE THREAD

**Issue by hepcat72** [libwww-perl/libwww-perl#118]
I may just have a totally messed up system, but anyone have any suggestions how to deal with this:

```
% make test
PERL_DL_NONLAZY=1 /usr/bin/perl "-MExtUtils::Command::MM" "-e" "test_harness(0, 'blib/lib', 'blib/arch')" t/*.t t/base/*.t t/local/*.t t/robot/*.t
t/base/default_content_type.t .. 1/10 
#   Failed test 'put HASH(0x7fb0a2195650)'
#   at t/base/default_content_type.t line 39.
#          got: 'PUT http://www.example.com
# User-Agent: foo/0.1
# Content-Type: application/json
# HASH(0x7fb0a2195650): 
# 
# '
#     expected: 'PUT http://www.example.com
# User-Agent: foo/0.1
# Content-Length: 7
# Content-Type: application/x-www-form-urlencoded
# 
# cat=dog
# '

#   Failed test 'put ARRAY(0x7fb0a2195710)'
#   at t/base/default_content_type.t line 39.
#          got: 'PUT http://www.example.com
# User-Agent: foo/0.1
# Content-Type: application/json
# ARRAY(0x7fb0a2195710): 
# 
# '
#     expected: 'PUT http://www.example.com
# User-Agent: foo/0.1
# Content-Length: 7
# Content-Type: application/x-www-form-urlencoded
# 
# cat=dog
# '
Not a SCALAR reference at /System/Library/Perl/Extras/5.18/HTTP/Message.pm line 163.
# Looks like you planned 10 tests but ran 6.
# Looks like you failed 2 tests of 6 run.
# Looks like your test exited with 2 just after 6.
t/base/default_content_type.t .. Dubious, test returned 2 (wstat 512, 0x200)
Failed 6/10 subtests 
t/base/protocols.t ............. ok   
t/base/proxy.t ................. ok   
t/base/simple.t ................ ok   
t/base/ua.t .................... ok     
t/local/autoload-get.t ......... ok   
t/local/autoload.t ............. ok   
t/local/get.t .................. ok   
t/local/http.t ................. ok     
t/local/httpsub.t .............. ok   
t/local/protosub.t ............. ok   
t/robot/ua-get.t ............... ok     
t/robot/ua.t ................... ok     

Test Summary Report
-------------------
t/base/default_content_type.t (Wstat: 512 Tests: 6 Failed: 2)
  Failed tests:  5-6
  Non-zero exit status: 2
  Parse errors: Bad plan.  You planned 10 tests but ran 6.
Files=13, Tests=198, 15 wallclock secs ( 0.11 usr  0.05 sys +  2.37 cusr  0.45 csys =  2.98 CPU)
Result: FAIL
Failed 1/13 test programs. 2/198 subtests failed.
make: *** [test_dynamic] Error 255
```

**Comment by genio** [comment]
Hi @hepcat72, unfortunately we created some tests that depend on a slightly newer version of HTTP::Message (at least 6.07). We are currently fixing this oversight and will release a fix soon.  Until then, it's safe to ignore the failed test or you can update HTTP::Message.

**Comment by oalders** [comment]
This should be resolved 6.22 which has just been released.
